sdk/healthdataaiservices/azure-health-deidentification/spotbugs-exclude.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
 <FindBugsFilter xmlns="https://github.com/spotbugs/filter/3.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                 xsi:schemaLocation="https://github.com/spotbugs/filter/3.0.0 https://raw.githubusercontent.com/spotbugs/spotbugs/3.1.0/spotbugs/etc/findbugsfilter.xsd">
+  <Match>
+    <Bug pattern="UPM_UNCALLED_PRIVATE_METHOD"/>
```

Comment:
Why do we have an unused private method?

Reply:
removed